### PR TITLE
timer fragment added with task view fragment

### LINKED ISSUE #22: fragment inflating problem
* layout overlapping issue

image

### PR DESCRIPTION
**Pull request by sajjad-h**

### CODE REVIEW

**Comment by mehedihasan88** [comment]
* UI seems less expressive now
* If I press go back from task list view , first the timer goes off , then again pressing go back  exits from the page- which should not be expected

File: TeamTracker/app/src/main/java/com/example/teamtracker/fragments/TaskViewFragment.java [hunk 1]
```diff
@@ -54,15 +52,9 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
...
-            @Override
-            public void onClick(View view) {
-                Fragment fragment = new TimerFragment(project);
-                FragmentManager fragmentManager = getActivity().getSupportFragmentManager();
-                fragmentManager.beginTransaction().replace(R.id.fragment_container, fragment).addToBackStack("TaskViewTimer").commit();
-            }
-        });
+        Fragment fragment = new TimerFragment(project);
+        FragmentManager fragmentManager = getActivity().getSupportFragmentManager();
+        fragmentManager.beginTransaction().replace(R.id.timer_fragment_container, fragment).addToBackStack("TimerFragment").commit();
```
Comment (mehedihasan88) [inline TeamTracker/app/src/main/java/com/example/teamtracker/fragments/TaskViewFragment.java]: removing the addToBAckStack solves the problem of < going back from task list to project list > problem .. @sajjad-h

**Comment by sajjad-h** [comment]
> * UI seems less expressive now
> 
>     * If I press go back from task list view , first the timer goes off , then again pressing go back  exits from the page- which should not be expected

problem solved. can you check now, please?

**Review by mehedihasan88 — APPROVED** [review]
good work

**Review by mehedihasan88 — APPROVED** [review]
Good Work!keep it up..

**Pull request merged**